tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -300,12 +300,2652 @@
     "scrolled": true
    },
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/manoel/projects/manoelmarques/qiskit-terra/qiskit/utils/measurement_error_mitigation.py:44: DeprecationWarning: Back-references to from Bit instances to their containing Registers have been deprecated. Instead, inspect Registers to find their contained Bits.\n",
```

Comment:
Do we want to hide these deprecation warnings for the notebook? Or maybe update `measurement_error_mitigation`?

Reply:
The deprecation msgs are eliminated in Qiskit/qiskit-terra#6503